types/js-cookie/index.d.ts

```diff
@@ -69,18 +69,6 @@ declare namespace Cookies {
          */
         get(): {[key: string]: string};
 
-        /**
```

Comment:
Were these removed in 3.0?  If so, does the version number at the top need to be updated?

Reply:
You're right. I was confused by the fact that version umber was bumped two commits after. Will fix that ASAP.

Reply:
I bumped the version number, but I feel like this should wait the end of the beta before being merged.

Should I close the PR and resub;it it later ?

Reply:
Also, the generic on `CookieStatic` should expand `string` and not `object` as set doesn't accept array or literals anymore.

Reply:
If you like, you can mark it as Draft until you're ready to go.

Reply:
Well, I have another one to submit for branch 2.x, so this one will be rebased. It will be more simple to just resubmit it later.

Thank you for the review!